content/en/docs/deprecations/aliases_in_comparason_operators.md

```diff
@@ -44,6 +44,6 @@ flags: {
 }
 ```
 
-More `default.js` configuration file information can be found here: [Project Configuration](/docs/developer_guide/project/configuration/project_configuration/#flags)
+For more detailed information on the configuration options, see [Project configuration](/docs/developer_guide/project/configuration/project_configuration/#flags).
```

Comment:
Unfortunately, I had to make this change. The link title was wrong, and the text was meh, and we're referring default.js, which isn't always strictly that file, and we're copy/pasting it in quite a few places. I considered using a short-code, but decided against it. I did limit the changes to this reference only.